### PR TITLE
gcc@8: use install-strip for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**
Aligning code with changes made in #9031

### CODE REVIEW

File: Formula/gcc@8.rb [hunk 1]
```diff
@@ -162,17 +156,6 @@ def install
     Dir.glob(man7/"*.7") { |file| add_suffix file, version_suffix }
     # Even when we disable building info pages some are still installed.
     info.rmtree
-
-    unless OS.mac?
-      # Remove files conflicted with gcc formula.
-      rm_f [lib/"libgcc_s.so.1", lib/"libstdc++.so.6"]
```
Comment (xu-cheng) [inline Formula/gcc@8.rb]: This should not be removed.

Comment (sjackman) [reply Formula/gcc@8.rb]: Other `gcc@…` formulae had the block that created the symlinks, but not the block that removed the symlinks, so the resulting packages conflicted with the unversioned `gcc` formula.

An alternative fix is to use `-static-libstdc++` to build the bootstrap compiler.

Comment (sjackman) [reply Formula/gcc@8.rb]: This issue does not appear to affect building these formulae on CircleCI, which works as expected.

Comment (xu-cheng) [reply Formula/gcc@8.rb]: Sorry, I am a little confused about where these symlink com from. I thought it comes from the gcc itself. If it were coming from the formula (which has now been removed in this PR), then it should be fine now.

Comment (iMichka) [reply Formula/gcc@8.rb]: Yes, I was confused too, but as these were added by us manually, and I am removing them from the gcc 8 code above, we can remove this line too.

**Pull request closed without merging**